js/src/wp-seo-admin-global.js

```diff
@@ -293,10 +293,83 @@
 		} );
 	}
 
+	/**
+	 * Make tables scrollable.
+	 *
+	 * Usage: see related stylesheet.
+	 *
+	 * @returns {void}
+	 */
+	function scrollableTables() {
```

Comment:
The name doesn't really explain what the code does. Something like `createScrollableTables` would better describe what is happening here.